forge-gui/res/cardsfolder/upcoming/adrestia.txt

```diff
@@ -0,0 +1,17 @@
+Name:Adrestia
+ManaCost:3
+Types:Legendary Artifact Vehicle
+PT:4/3
+K:Islandwalk
+T:Mode$ Attacks | ValidCard$ Card.Self | CheckSVar$ X | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME attacks, if an Assassin crewed it this turn, draw a card. CARDNAME becomes an Assassin in addition to its other types until end of turn.
+SVar:TrigDraw:DB$ Draw | NumCards$ 1 | SubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | Defined$ Self | Types$ Assassin
+T:Mode$ BecomesCrewed | ValidVehicle$ Card.Self | Execute$ RememberCrew | Static$ True
+SVar:RememberCrew:DB$ PumpAll | ValidCards$ Creature.TriggeredCrew | RememberPumped$ True
```

Comment:
the check is based on LKI, so you should only remember Assassins here (instead of checking the type later)